cli/Cargo.toml

```
@@ -14,9 +14,12 @@ name = "jstime"
 path = "main.rs"
 
 [dependencies]
-jstime_core = { path = "../core", version = "^0.12.5-alpha.0"}
+jstime_core = { path = "../core" }
```

```suggestion
jstime_core = { path = "../core", version = "^0.12.5-alpha.0"}
```
We need to keep the version number here or we break publication. The release infrastructure we have automates keeping this up to date during publication